curator-recipes/src/main/java/org/apache/curator/framework/recipes/cache/ChildData.java

```diff
@@ -126,12 +126,7 @@ public Stat getStat()
      */
     public byte[] getData()
     {
-        return data.get();
-    }
-
-    void clearData()
-    {
-        data.set(null);
+        return data;
```

Comment:
clearData() needs to remain. PathChildrenCache uses it to release memory. The data is needed temporarily and is cleared afterwards.